### PR TITLE
Fujitatomoya/GitHub workflow build verify

### PR DESCRIPTION
**Pull request by fujitatomoya**
address https://github.com/Haivision/srt-prometheus-exporter/issues/4

### CODE REVIEW

**Comment by fujitatomoya** [comment]
@Windrow @Windrow14 requesting review

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+
+    name: build-${{matrix.os}}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
```
Comment (fujitatomoya) [inline .github/workflows/build.yml:21]: once we support ubuntu-latest which is ubuntu-22.04, we can just add it into this list.

File: sample/Makefile [hunk 2]
```diff
@@ -22,23 +22,23 @@ LIBS = -lsrtexp
...
-$(SAMPLE_APP_SRT_DESTINATION_C): $(SRCDESTINATION)
-	$(CC) -o $@ $(CFLAGS) $(SRCDESTINATION) $(LDFLAGS) $(LIBS)
+$(SAMPLE_APP_SRT_LISTENER): $(SRC_LISTENER)
+	$(CC) -o $@ $(CFLAGS) $(SRC_LISTENER) $(LDFLAGS) $(LIBS)
 
 clean:
-	-rm $(SAMPLE_APP_SRT_SOURCE_C)
-	-rm $(SAMPLE_APP_SRT_DESTINATION_C)
+	-rm $(SAMPLE_APP_SRT_CALLER)
+	-rm $(SAMPLE_APP_SRT_LISTENER)
```
Comment (fujitatomoya) [inline sample/Makefile:42]: these changes are required because of https://github.com/Haivision/srt-prometheus-exporter/issues/12

Comment (Windrow14) [reply sample/Makefile:42]: I'm sorry about the troubles, but the real problem is, I forgot to change c files' name.  
`sample_srtcaller.c` ->`sample_srtsource.c`  
`sample_srtlistener.c`->`sample_srtdestination.c`

Comment (Windrow14) [reply sample/Makefile:42]: We can merge this PR first, then reopen https://github.com/Haivision/srt-prometheus-exporter/issues/12. I'll pull that again.

**Pull request merged**